app/clients/publishing_api/recommended_link_client.rb

```diff
@@ -0,0 +1,38 @@
+module PublishingApi
+  class RecommendedLinkClient
```

Comment:
Something to ponder. Is a separate client needed for each "thing" that needs to update publishing-api? Will more than recommended links need to be pushed to to publishing-api?

If yes, then I believe the only information that's different is the payload, so if the pattern is that all of the models using this client have a `to_publishing_api_content_item` method on them, then the name of this class and the object passed to it can be generic.

Reply:
Good shout - it's only recommended links, with no plan to have anything else (that I can think of), but it probably makes more sense to name the client after the _target_ rather than the source of thing it deals with 👍